src/views/base/navigation/header.module.scss

```
@@ -72,7 +78,7 @@ header {
 		margin-right: var(--header_logo_padding-end);
 		line-height: 125%;
 
-		color: var(--primary_default);
+		color: var(--primary_medium);
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)